onken/public/urls.py

```diff
@@ -12,4 +22,6 @@
 
     path('login', django_cas_ng.views.login),
     path('logout', django_cas_ng.views.logout),
+
+    url(r'^api/', include(router.urls)),
```

Comment:
Please use the `path` function here; [`url` will be deprecated soon](https://docs.djangoproject.com/en/2.0/ref/urls/#url).